src/main/java/mlib/tool/SysMsg.java

```diff
@@ -17,10 +17,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-package mSearch.tool;
+package mlib.tool;
 
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
+import mlib.tool.SysMsg;
 
 public class SysMsg {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)